### ISSUE TITLE
Disable smooth scrolling feature

### ISSUE THREAD

**Issue by jamesWalker55** [black7375/Firefox-UI-Fix#225]
**Is your feature request related to a problem?**
I have my own scroll settings set in about:config, after installing the theme my scroll settings are overridden with the ones defined in the theme. My own scroll settings are gone and I need to set them again every time I open the browser.

**The solution I'd like**
Remove or allow disabling the included smooth scrolling settings, I don't think a UI fix should affect the behaviour of the browser or override user settings.

**Comment by dokax** [comment]
>I don't think a UI fix should affect the behaviour of the browser or override user settings.

I feel the same.

**Comment by MarmotMentality** [comment]
Yes I agree as well. While I'm sure some people like it, I don't. The smoothscroll stuff should be a different project. 

I went into about:config and reverted all the changes it made and went back to the settings I already had. Then I deleted the smoothscroll section in user.js so it won't come back next time I open Firefox. Still love this mod but this was a bit annoying.

**Comment by black7375** [comment]
Well. It seems to be a feature that can be disliked.

However, scrolling itself can still be targeted as a UX improvement target.
There's no reason to give up on Better default.
The problem is that the current scrolling settings can be annoying.

Has anyone tried the following settings?
https://github.com/black7375/Firefox-UI-Fix/wiki/Tips#only-sharpen-scrolling
It's a small change, but it improves responsiveness.

**Comment by jamesWalker55** [comment]
I agree that scrolling can be seen as a UX target, but the issue is that the theme overrides any custom user settings related to scrolling. If a user defined some scroll settings in `about:config`, those settings will be lost after installing the theme.

**Comment by plomari** [comment]
Unfortunately, I copied this repo's user.js blindly into my firefox profile, which caused smooth scrolling to be forced disabled somehow. "general.smoothScroll" was of course set to false, and user.js didn't change it, yet smooth scrolling happened when using the keyboard keys. Mouse scrolling was unaffected.

Only searching for "smooth" in about:config and manually reverting all listed settings (and then setting smoothScroll to false again) helped. It appears at least one of the settings changed by user.js ignores the global smoothScroll setting if set to a non-default. (As I said, "general.smoothScroll" was always set to false, except when I temporarily reverted it to its default along with other settings.)

I **REALLY** did not appreciate this and it took me some time to figure out. Next time, I will be more careful, and copy userChrome.css only. Actually, I hope there will be no next time, and I will have found a replacement for this horrible browser, because it's unacceptable that I have to spend an hour or longer to fix their worthless GUI changes every time. (I had to make some other changes, like moving the tab bar below the address bar. This project provides some CSS fragments to do it, I'm not affiliated with it: https://github.com/Aris-t2/CustomCSSforFx )

Also, could it be that the CSS changes make the main menu bar show icons besides the text? It's what you want for submenus and popup menus, but not the main menu. I'm not sure whether this repo's CSS does this, or if something else caused it.

**Comment by kanlukasz** [comment]
Yes, I feel the same too.
Smoothscroll stuff should be a different project

**Comment by black7375** [comment]
First of all, smooth scrolling will be removed in v4.5.

However, scrolling itself is a known shortcoming of Firefox and is something that needs improvement.
If you have an appropriate profile value, please share.

**Comment by black7375** [comment]
So is it better to specify the default value in `user.js`?

**Comment by Name626** [comment]
I have the same issue. After changing the scroll settings to what I prefer, if I close and open the browser, it gets reset to the Lepton smooth scrolling settings.

**Comment by remyabel2** [comment]
If you remove a value from `user.js`, it does not get reset in `prefs.js`. However, if you keep a value in `user.js` and restart the browser, the values from `user.js` are automatically applied. So one would have to manually reset the config to default in `about:config` after removing the value from `user.js`. Ideally Firefox-UI-Fix should change as little preferences as possible for this exact reason.
